README.rst

```diff
@@ -3,8 +3,33 @@ adb-utils
 
 Utility and Service scripts for the `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_.
 
+adb-utils provides following content to ADB Vagrant box
+
+* OpenShift as a service
+
+  adb-utils uses OpenShift docker containers and provides systemd service unit file which inludes some setup scripts for the purpose. With the systemd unit file, OpenShift can be configured and run in ADB with just "systemctl start openshift" command.
+
+* SCCLI
+
+The idea behind providing SCCLI is to provide a way for users who just want to do vagrant ssh in to the ADB vagrant box and do their work. With this CLI they can setup required providers e.g. single node Kubernetes , OpenShift , Mesos marathan etc with single CLI commands.
+
+With Release 1.0 we are providing interface for setting up OpenShift or single node Kubernetes.
+
+The usage of sccli is as below::
+
+       $ sccli --help
+
+       usage: sccli [service_name] || [clean]
+       List of possible service_name:
+                k8s punchiest
+
+
+This is packaged as a RPM and included in `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_ version 1.7.0 or later
```

Comment:
These utilities and unit files are packaged as an RPM and included in `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_ version 1.7.0 or later.